lib/pdk/telemetry.rb

```diff
@@ -0,0 +1,41 @@
+require 'securerandom'
+require 'pdk/telemetry/util'
+require 'pdk/telemetry/client/google_analytics'
+require 'pdk/telemetry/client/noop'
+
+module PDK
+  module Telemetry
+    def self.build_client
+      config_file = File.expand_path('~/.puppetlabs/bolt/analytics.yaml')
```

Comment:
nit: Could you add a note here that we'll be reviewing this filename/location.  It's not exactly intuitive to disable PDK telemetry in a directory called bolt.

Reply:
Yeah, probably worth at least a `TODO` here and maybe a reference to the ticket where we plan to move the config.

Reply:
Yep, I've added a TODO with the ticket ID for the task to rewrite the config loading.